fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/FedoraToOCFLObjectIndexImpl.java

```diff
@@ -35,10 +46,36 @@
  */
 @Component
 public class FedoraToOCFLObjectIndexImpl implements FedoraToOCFLObjectIndex {
+
+    private Path indexFilePath = getProperty("fcrepo.ocfl.work.dir") == null ?
+            Paths.get(JAVA_IO_TMPDIR, "fcrepo.ocfl.work.dir", "fedoraToOcflIndex.tsv") :
```

Comment:
Make fedoraToOcflIndex.tsv a constant as well.

Reply:
I just made a constant of the FILE with the text "fedoraToOcflIndex.tsv", I could make that String a constant and use it in the other constant.

But that seemed like overkill as I feel that this will go away once we have a non-in-memory index.

Reply:
👍